File: Source/OregonTilth.Models/DataTransferObjects/Workbook/Forms/CropSpecificInfo/CropSpecificInfoSummaryDto.cs
namespace OregonTilth.Models.DataTransferObjects
{
    public class CropSpecificInfoSummaryDto 
    {
        public int? CropSpecificInfoID { get; set; }
        public int WorkbookID { get; set; }
        public CropSummaryDto Crop { get; set; }
        public TpOrDsTypeDto TpOrDsType { get; set; }
        public decimal RowsPerStandardWidth { get; set; }
        public int DripTapeRowsPerStandardWidth { get; set; }
        public decimal? InRowSpacing { get; set; }
        public decimal? SeedCostPerStandardUnitOfSpace { get; set; }
        public decimal? TransplantProductionCostOutsourced { get; set; }
    }
}

File: OregonTilth.Web/src/app/shared/models/forms/crop-specific-info/crop-specific-info-summary-dto.ts
import { CropDto } from "../../generated/crop-dto";
import { TpOrDsTypeDto } from "../../generated/tp-or-ds-type-dto";

export class CropSpecificInfoSummaryDto {
    CropSpecificInfoID: number;
    WorkbookID: number;
    Crop: CropDto;
    TpOrDsType: TpOrDsTypeDto;
    RowsPerStandardWidth: number;
    DripTapeRowsPerStandardWidth: number;
    InRowSpacing: number;
    SeedCostPerStandardUnitOfSpace: number;
    TransplantProductionCostOutsourced: number;
    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}
